product_list.php

```diff
@@ -0,0 +1,93 @@
+<?php 
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+require_once 'config/constants.php';
+
+// Connecting to DB
+$conn = mysqli_connect(SERVERNAME, USERNAME, PASSWORD, DBNAME);
+
+// Handled case if connection failed
+if (!$conn) {
+    die("Connection failed: " . mysqli_connect_error());
+}
+
+if(isset($_GET['delete_id']) ) {
+    $sql_delete="DELETE FROM `products_list` WHERE `products_list`.`id` ='". $_GET['delete_id']."'";
+    mysqli_query($conn, $sql_delete);    
+} 
+
+
+
+$sql_getproducts="SELECT pl.id,pc.name as category_name,pl.image,pl.name as product_name,pl.amount,"
+        . "pl.description,pl.created_date FROM products_list pl "
+        . "JOIN products_category pc ON pl.category=pc.id WHERE pl.user_id=33 "
+        . "ORDER BY pl.created_date DESC" ;
+echo $sql_getproducts;
+
+$products = mysqli_query($conn, $sql_getproducts);
+
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+    <!--head-->
+    <head>
+        <title>QuickSeller:Products List</title>  
+        <?php
+           require_once 'templates/header.php';     
+        ?>
+    </head>
+
+
+<body >
+
+    <!-- Include the navigation bar -->
+    <?php require_once 'templates/navigation.php'; ?>
+
+    <section>
+        <?php if(isset($_GET['message'])) {?>
+            <div class='alert-success'> <?php echo $_GET['message']?> ! </div>
+        <?php } ?>
+    </section>
+    
+
+    <div class="container table-responsive">
+      <h2>Your Products</h2>
+    
+      <table class="table table-bordered table-condensed" >
+        <thead>
+          <tr>
+            <th>Category</th>
+            <th>Image</th>
+            <th>Name</th>
+            <th>Amount</th>
+            <th>Description</th>
+            <th>Uploaded on</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+            <?php
+                if(mysqli_num_rows($products)>0){
```

Comment:
Handling only tbody contents if no products present.
Please handle complete table, if no products present do not show table.
Can show some message like 'No products found to list.'